com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -16,7 +16,14 @@ The Render Pipeline Debugger window.
 
 ## Using the Render Pipeline Debugger window
 
-To open the Render Pipeline Debugger window in the Editor, go to  **Window > Analysis > Rendering Debugger**. You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
+To open the Render Pipeline Debugger window in the Editor, you must first enable **Runtime Debug Shaders** in **HDRP Global Settings**:
```

Comment:
Change "Render Pipeline Debugger window" to "Rendering debugger" in all instances, as per our earlier fixes. This is in an attempt to mitigate merge conflicts.